src/software/ai/navigator/obstacle/obstacle_factory_test.cpp

```diff
@@ -107,7 +159,7 @@ TEST_F(ObstacleFactoryTest, slow_moving_robot_obstacle)
     Vector velocity(0.0007, 0.004);
     Angle orientation(Angle::fromRadians(2.2));
     AngularVelocity angular_velocity(AngularVelocity::fromRadians(-0.6));
-    Circle expected(origin, 0.041569219381653054);
+    Circle expected(origin, 0.207);
```

Comment:
are these value changes due to bugfixes or some other change?

Reply:
These are bug fixes. This has been broken for a while but before we had the visualizer drawing obstacles, it was hard to debug.